locales/en-US/server.ftl

```diff
@@ -78,14 +102,20 @@ shotPageDeleteButton
   .title = Delete this shot permanently
 shotPageAbuseButton
   .title = Report this shot for abuse, spam, or other problems
-shotPageMyShots = My Shots
 shotPageDownloadShot
   .title = Download the shot image
 shotPageDownload = Download
 // Note: do not translate 'Firefox Screenshots' when translating this string
 shotPageScreenshotsDescription  = Firefox Screenshots made simple. Take, save and share screenshots without leaving Firefox.
 shotPageUpsellFirefox = Get Firefox now
 shotPageDoesNotExpire = does not expire
+shotPageDMCAMessage = This shot is no longer available due to a third party intellectual property claim.
```

Comment:
@johngruen Are we going to want the community localizers to handle the DMCA legalese, or do we need to handle this via a paid service?